src/include/ipc4/base_fw.h

```diff
@@ -359,6 +359,12 @@ enum ipc4_fw_config_params {
 	IPC4_TELEMETRY_BUFFER_SIZE = 24,
 	/* HW version information  */
 	IPC4_BUS_HARDWARE_ID = 25,
+	/* Total memory size available in debug window */
```

Comment:
is this now aligned with reference IDs ?

Reply:
yes, sync with latest ID